src/actions/BitPayActions.js

```diff
@@ -225,6 +221,20 @@ export async function launchBitPay(
     onDone: (error: Error | null, edgeTransaction?: EdgeTransaction) => {
       if (error) showError(`${s.strings.create_wallet_account_error_sending_transaction}: ${error.message}`)
       Actions.pop()
+      if (!error && edgeTransaction) {
+        fetchBitPayJsonResponse(uri, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/payment',
+            'x-paypro-version': '2'
+          },
+          body: JSON.stringify({
+            chain: requestCurrencyCode,
+            currency: requestCurrencyCode,
+            transactions: [{ tx: signedHex, weightedSize: signedHex.length / 2 }]
```

Comment:
It appears that the `signedHex` which is transmitted here results in a different txid than the one Edge displays in the payment history.

That could possibly be the cause of the tx-mempool-conflict I am receiving when pushing transactions to the blockchair API.